lib/smart_answer_flows/next-steps-for-your-business.rb

```diff
@@ -0,0 +1,133 @@
+# ======================================================================
+# The flow logic.
+# ======================================================================
+
+module SmartAnswer
+  class NextStepsForYourBusinessFlow < Flow
+    def define
+      name "next-steps-for-your-business"
+      start_page_content_id "4d7751b5-d860-4812-aa36-5b8c57253ff2"
+      flow_content_id "981e0708-9fa5-42fb-baf5-ee5630a9b722"
+      status :draft
+      use_session true
+      # ======================================================================
+      # Available input types:
+      # ======================================================================
+      # - Checkbox
+      # - Country select
+      # - Date
+      # - Money
+      # - Radio
+      # - Postcode
+      # - Salary
+      # - Value (text)
```

Comment:
Happy to leave this in if it's helpful (it should also be listed in the Smart Answer docs) - otherwise can we remove it to save some scrolling?